pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,76 @@
-`--non-validating-full-node=true` flag. Otherwise, the application will require that gRPC be enabled.
-
-
-## Staying up-to-date with the Latest State
-
-### Connecting to Healthy Peers
-
-In order for the full node to have the latest state, it needs to connect to peers which have the latest state Use the following links to get a list of live peers which have the latest state
-
-https://services.lavenderfive.com/mainnet/dydx#live-peers
+<!-- do these need to be in square brackets? it seems like thats how toml arrays work -->
 
-https://polkachu.com/live_peers/dydx
+## Save an Address Book
+Download the latest `addrbook.json` file, which stores configuration details that help your node efficiently connect to peers in its network. You can download an up-to-date address book file from one of the following services:
+- https://polkachu.com/addrbooks/dydx
+- https://services.lavenderfive.com/mainnet/dydx#latest-addrbook
+- https://autostake.com/networks/dydx/
 
-Update `persistent_peers`  in the config.toml file to include a randomly selected list of 5 peers from the list of live peers obtained using the links above
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-### Snapshots
+## Back Up Your Node
+> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed and can skip this section.
 
-Snapshots contain a compressed copy of the chain data which allow the full node to bootstrap to a recent state in the blockchain. A list of snapshot services can be found [here](../infrastructure_providers-network/resources.mdx#snapshot-service).
+Your full node needs a backup plan to replay the history of the network in case it falls out of sync. You can back up your node in one of two ways:
 
-### State sync (Alternative to snapshots)
+1. Using a **snapshot**, which contains a compressed copy of the application state at a point in time, saved on the system that your node runs on. If your node falls out of sync, the snapshot allows the node to quickly recover to the application state when your snapshot was taken. This speeds up the syncing process significantly. To use a snapshot to back up your full node, install a snapshot from a [snapshot service](../infrastructure_providers-network/resources.mdx#snapshot-service).
 
-State Sync enables a new node to join the network by obtaining a snapshot of the application state from a state sync node at a recent height. This eliminates the need to fetch and replay all historical blocks. A list of state sync services with instructions are listed below
+2. Using **state sync**, which enables a your node to obtain a snapshot of the application state from a state sync node. This speeds up the syncing process significantly. To use state sync to back up your full node, follow the instructions from a state sync service from the list below:
```

Comment:
I want to drill into the differences between these two. My understanding:

- snapshot route is usually faster (because the snapshot is already available?)
- state sync will still ultimately use a snapshot, but will source it from a peer node

Is that correct? what else should we highlight about these two methods?

Reply:
@affanv14 are you able to answer this?

Reply:
@zachfc your assumption is correct. that state sync ultimately uses a snapshot but state sync has the latest state whereas for snapshot you still need to replay a few block to catch up to latest state from snapshot state

refer https://services.lavenderfive.com/mainnet/dydx/statesync